ccdproc/tests/test_combiner.py

```diff
@@ -100,6 +100,16 @@ def test_weights_shape(ccd_data):
     with pytest.raises(ValueError):
         c.weights = ccd_data.data
 
+def test_1Dweights():
+    ccd_list = [CCDData(np.zeros((10, 10)), unit=u.adu),
+                CCDData(np.zeros((10, 10)) - 1000, unit=u.adu),
+                CCDData(np.zeros((10, 10)) + 1000, unit=u.adu)]
+        
+    c = Combiner(ccd_list)
+    c.weights = np.array([1, 5, 10])
+    ccd = c.average_combine()
+
+
```

Comment:
Sorry for the nitpicky comment: We're mostly using PEP8 with 2 empty lines between functions. That means it's one empty line too few before the function but one too much after. Could you fix that?

Reply:
Please don't "resolve conversation" when they are in fact not resolved (either in code or by comment).